samples/tfm/tfm_psa_template/README.rst

```diff
@@ -119,6 +119,7 @@ This sample supports firmware update of both the application and TF-M, and the s
 
 The firmware update process requires signature verification keys in order to sign the images used in the firmware update process.
```

Comment:
Could we say the following instead?
The firmware update process requires signature verification keys in order to sign the images used in the process.

Reply:
Sure